File: CRM/src/app/shared/eHospital_DongNai_A/ztmp_BCDP_003.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ztmp_BCDP_003 } from './ztmp_BCDP_003.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ztmp_BCDP_003Service extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'STT', 'TenKho', 'DauKy', 'TongLuongNhap', 'TongLuongXuat'];

    List: ztmp_BCDP_003[] | undefined;
    ListFilter: ztmp_BCDP_003[] | undefined;
    FormData!: ztmp_BCDP_003;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ztmp_BCDP_003";
    }
}

File: Service_eHospital_DongNai_A/Implement/ztmp_BCDP_003Service.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class ztmp_BCDP_003Service : BaseService<ztmp_BCDP_003, Iztmp_BCDP_003Repository>
    , Iztmp_BCDP_003Service
    {
    private readonly Iztmp_BCDP_003Repository _ztmp_BCDP_003Repository;
    public ztmp_BCDP_003Service(Iztmp_BCDP_003Repository ztmp_BCDP_003Repository) : base(ztmp_BCDP_003Repository)
    {
    _ztmp_BCDP_003Repository = ztmp_BCDP_003Repository;
    }
    }
    }

